core/src/main/java/com/group5/core/controllers/GameSliceSequenced.java

```diff
@@ -11,7 +11,7 @@
 /**
  * GameSlice is a 2Dimensional rectangle in which objects are placed.
  */
-public abstract class GameSlice {
+public abstract class GameSliceSequenced {
```

Comment:
I'd say SequencedGameSlice would be more in line with standard Java class naming conventions